_episodes_rmd/07-vector-shapefile-attributes-in-r.Rmd

```diff
@@ -356,7 +356,7 @@ Let's add a legend to our plot. We will use the `road_colors` object
 that we created above to color the legend. We can customize the
 appearance of our legend by manually setting different parameters.
 
-```{r add-legend-to-plot }
+```{r add-legend-to-plot fig.cap="Roads and trails in the study area using thicker lines than the previous figure."}
```

Comment:
```suggestion
```{r add-legend-to-plot, fig.cap="Roads and trails in the study area using thicker lines than the previous figure."}
```